### PR TITLE
Surf & Paddle  and fixing Chess

### PR DESCRIPTION
**Pull request by carolinapowers**
List of known issues:
- [ ] Sidebar text needs padding.
- [ ] Hover state on most popular posts need improvement.
- [ ] Footer.  
- [ ] Redo it with completely new code since I didn't have time to get to it.

### CODE REVIEW

File: surf+paddle/index.html [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+                        <li><a href='/about'>ABOUT</a></li>
+                        <li><a href='/search'>SEARCH<i class="fa fa-search"></i></a></li>
+                         </ul>
+                </nav>
+            
+               <div id="logo">
+               <img src="https://s3.amazonaws.com/uploads.hipchat.com/31349/1589699/2cBN2RS7COcTNq8/wave_small.png"><span id="reveal-logo">SURF & PADDLE CO. BLOG</span>
+           </div>   
+                
+                <h1>SURF & PADDLE CO.</h1>
```
Comment (mfees) [inline surf+paddle/index.html:42]: to make & you need to do `&amp;`
also you should keep your text lowercase and style it in css with 
text-transform: uppercase

**Comment by mfees** [comment]
:+1: 

it was hard to find the files to look at, maybe do different feature branches next but good job

File: surf+paddle/index.html (hunk 1 above)
Comment (carolinapowers) [inline surf+paddle/index.html:42]: thanks Maddie! :+1:

**Comment by jaredstevick** [comment]
:+1:

**Pull request merged**